testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_reserve_properties.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from CBLClient.Document import Document
+from keywords import document
+from keywords import attachment
+
+
+@pytest.mark.listener
+@pytest.mark.properties
+def test_reserve_property(params_from_base_test_setup):
+    """
+    @summary:
+    1. Create docs in CBL using the _id
+    2. Make sure _id is not supported while creating the doc .
+    3. In future releases _exp, _attachments will not be supported too, then we need to update the same test
+    """
+
+    base_url = params_from_base_test_setup["base_url"]
+    liteserv_version = params_from_base_test_setup["liteserv_version"]
+    db = params_from_base_test_setup["db"]
+    cbl_db = params_from_base_test_setup["source_db"]
+    if liteserv_version < "3.0.0":
+        pytest.skip('This test cannot run with CBL version below 3.0.0')
+    channel = ["Replication-1"]
+    doc_id = "doc_1"
+    documentObj = Document(base_url)
+    doc_body = document.create_doc(doc_id=doc_id, content="doc1", channels=channel, expiry=2,
+                                   attachments=attachment.generate_2_png_10_10())
```

Comment:
could you please add some comments about how document.create_doc would react? (while cbl=False or no value set for cbl) otherwise, reading this code will not get any sense how _id is involved, unless go to check document_create_doc function definition.

Reply:
correct, Updated as per your comment

Reply:
great